documentation/modules/rn-2.6.adoc

```diff
@@ -58,6 +58,22 @@ The {project-short} web console can no longer download logs. With this update, y
 
 This part provides features and enhancements introduced in {project-full} 2.6:
 
+.Preservation of static IPs in migrations from vSphere
+
+You can now preserve static IPs of VMs that are installed with a Windows operating system and are migrated from vSphere. link:https://issues.redhat.com/browse/MTV-1148[(MTV-1148)]
+
+.Support for migrating LUKS-encrypted devices in migrations from vSphere
+
+You can now migrate devices that have been encrypted by Linux Unified Key Setup (LUKS) when you create a vSphere migration plan. link:https://issues.redhat.com/browse/MTV-831[(MTV-831)]
```

Comment:
(1) the decryption applies only to disks so I'd replace `devices` with `virtual disks`
(2) the user could select such VMs when creating a migration plan before so I'd drop the `when you create a vSphere migration plan` part
how about:
```suggestion
You can now migrate VMs with virtual disks that have been encrypted by Linux Unified Key Setup (LUKS) using cold migrations from a vSphere provider. link:https://issues.redhat.com/browse/MTV-831[(MTV-831)]
```

Reply:
Fixed, with mild rewrite.